docs/pages/more/create-expo.mdx

```diff
@@ -21,7 +21,7 @@ To create a new project, run the following command:
 
 </Tab>
 
-<Tab label="Yarn">
+<Tab label="yarn">
```

Comment:
"Yarn" name should be capitalized:
* https://yarnpkg.com/
* https://github.com/yarnpkg/berry

Reply:
<img width="316" alt="image" src="https://github.com/user-attachments/assets/db1d45f8-84d2-49c7-bbee-c529d004bc23" />

Reply:
And bun also regularly use the lowercase `bun` when referring to commands in their own docs 
<img width="809" alt="image" src="https://github.com/user-attachments/assets/43ee3ff3-6f4a-4315-b111-ce4864b74963" />

Reply:
Wordmark or CLI name =/= the official name. Wordmarks are usually used for marketing/presentation purpose, and npm pkg names or other registries have some restrictions which don't allow usage of proper name.

![Screenshot 2025-01-13 at 18 01 28](https://github.com/user-attachments/assets/a284cf9f-b5be-4f7f-9c9b-ae9a9e6e4a6f)
![Screenshot 2025-01-13 at 18 01 49](https://github.com/user-attachments/assets/d9fbb1c3-aca9-4fb3-bcd6-05c676d05da0)
![Screenshot 2025-01-13 at 18 02 00](https://github.com/user-attachments/assets/290deb43-2eb4-48e9-8dea-5c39a5bd280c)